manim/utils/ipython_magic.py

```diff
@@ -154,7 +154,12 @@ def construct(self):
                     logger.info("No output file produced")
                     return
 
-                local_path = Path(config["output_file"]).relative_to(Path.cwd())
+                # `media_embed` (as subpath, i.e. Path.cwd()) is unable to discover files if `media_dir` is absolute
+
+                local_path = Path(config["output_file"])
+                if not Path(config["output_file"]).is_absolute():
```

Comment:
```suggestion
                if not local_path.is_absolute():
```

Reply:
Hi, Benjamin.

So, currently manim allows users to specify the exact location for exporting/rendering media by setting the `config.media_dir` and retrieving the media for mime type rendering into ipython notebooks using `config.media_embed`

However, in the special case that the user requires to retrieve media from *outside* (not saved in a subdirectory) of the project root, i.e. far & beyond the directory tree in which the notebook file lives, there seems to be an issue with the line marked for deletion/replacement `local_path = Path(config["output_file"]).relative_to(Path.cwd())`

Specifically, the invokation `Path.cwd()` seems to be a restriction on the flexibility that `config.media_embed` promises to fulfill. The proposed changes provides a relaxation of this effect and is supposed to allow the user to retrieve media content from *outside* the project root (perhaps even on a different drive, such as in my case using wsl but rendering manim to a Windows directory, yet still desires to render the content into notebooks).  if the user specifies an absolute path for `config.media_dir` and `config.media_embed`, then ipython notebooks should be able to discover media under `/jupyter` at the specified directory, whether it be a parent of the project root, a subdirectory of the project root or a completely different drive altogether.

Please do not hesitate to reach out for any further clarification.

Best,
λ